integrations/parsely.php

```diff
@@ -13,24 +13,37 @@
  * @private
  */
 class ParselyIntegration extends Integration {
+	/**
+	 * Returns `true` if `Parse.ly` is already available via customer code. We will use
+	 * this function to prevent loading of integration again from platform side.
+	 */
+	public function is_loaded(): bool {
+		return class_exists( 'Parsely' ) || class_exists( 'Parsely\Parsely' );
+	}
+
 	/**
 	 * Loads the plugin.
 	 *
 	 * @private
 	 */
 	public function load(): void {
-		// Do not load plugin if already loaded by customer code.
-		if ( class_exists( 'Parsely\Parsely' ) ) {
-			return;
-		}
-
 		// Activating the integration via setting constant whose implementation is already
```

Comment:
```suggestion
		if ( $this->is_loaded() ) {
			return;
		}

		// Activating the integration via setting constant whose implementation is already
```

Reply:
Added in https://github.com/Automattic/vip-go-mu-plugins/pull/4839/commits/4bc3fdeda3547e91dcbc6c8656666e72049b7e2f